rdbms/cli.lsp

```diff
@@ -85,18 +80,22 @@
     ((equal cmd 'SAVE)   (process-save   db (read)))
     ((equal cmd 'HELP)   (help)))
   (cond
-    ((equal cmd 'EXIT) (write-line "bye"))
-    ((equal cmd 'LOAD)   (process-command (process-load      (read)) (read-command)))
-    ((equal cmd 'CREATE) (process-command (process-create db (read)) (read-command)))
-    ((equal cmd 'DROP)   (process-command (process-drop   db (read)) (read-command)))
-    ((equal cmd 'INSERT) (process-command (process-insert db (read)) (read-command)))
-    ((equal cmd 'UPDATE) (process-command (process-update db (read)) (read-command)))
-    ((equal cmd 'DELETE) (process-command (process-delete db (read)) (read-command)))
-    (t (process-command db (read-command)))))
+    ((equal cmd 'LOAD)   (process-load      (read)))
+    ((equal cmd 'CREATE) (process-create db (read)))
+    ((equal cmd 'DROP)   (process-drop   db (read)))
+    ((equal cmd 'INSERT) (process-insert db (read)))
+    ((equal cmd 'UPDATE) (process-update db (read)))
+    ((equal cmd 'DELETE) (process-delete db (read)))
+    (t db)))
+
+(defun process-command-loop(db)
+  (setq cmd (read-command))
```

Comment:
The feature of this project is not to use variables, so it's better to rewrite this part of code